tests/phpunit/tests/functions.php

```diff
@@ -1405,7 +1405,7 @@ public function data_wp_get_image_mime() {
 	 */
 	public function test_wp_getimagesize( $file, $expected ) {
 		if ( ! is_callable( 'exif_imagetype' ) && ! function_exists( 'getimagesize' ) ) {
-			$this->markTestSkipped( 'The exif PHP extension is not loaded.' );
+			$this->fail( 'The exif PHP extension is not loaded.' );
```

Comment:
Same thing again. Failing tests should only be done when the code under test doesn't run correctly. Not when the environment in which the tests are run does not comply with the desired environment.